packages/calling-stateful-client/src/CallSubscriber.ts

```diff
@@ -211,13 +209,15 @@ export class CallSubscriber {
   /* @conditional-compile-remove(close-captions) */
   private initCaptionSubscriber = (): void => {
     // subscribe to captions here so that we don't call captions when call is not initialized
-    if (_isTeamsMeetingCall(this._call) && this._call.state === 'Connected' && !this._captionsSubscriber) {
-      this._captionsSubscriber = new CaptionsSubscriber(
-        this._callIdRef,
-        this._context,
-        this._call.feature(Features.TeamsCaptions)
-      );
-      this._call.off('stateChanged', this.initCaptionSubscriber);
+    if (this._call.state === 'Connected' && !this._captionsSubscriber) {
```

Comment:
Having to read callState to understand if we can have access to a captions feature isn't great :/

@edwardlee-msft is this the documented way developers have to access captions - they must wait for the callstate to be connected _then_ access the captions API?

Ideally as a developer if I have a call object, regardless of the call state I could call `call.feature(Features.Captions).caption` and set up my subscriptions

Reply:
Yeah we can only subscribe to captions when the call is connected